patching/psp.md

```diff
@@ -0,0 +1,61 @@
+# Using Refresher to setup Allefresher on PSP/Vita
+
+Refresher has support for automatically deploying Allefresher onto a PSP or Vita, with the relavent configuration automatically filled out. This will let you connect to a custom server on LittleBigPlanet for the PSP (on Vita, this would be through Adrenaline).
+
+It also assumes that you've already downloaded Refresher. If you haven't, head on over to the [GitHub page](https://github.com/LittleBigRefresh/Refresher) to grab yourself a copy.
+
+![A Screenshot of Refresher's PSP patching screen](https://littlebigrefresh.github.io/Docs/patching/images/refresher-psp.png)
+
+## Step 1: Jailbreak your device and install a CFW
+
+We can't help you with this directly, but there are various guides out there you can go out and find. We personally recommend the guides on [pspunk](https://www.pspunk.com/psp-cfw/) and [vita.hacks.guide](https://vita.hacks.guide/) for PSP and Vita respectively.
+
+### PSP
+
+Try to use the latest firmware, and we recommend PRO C over other CFWs, as they have not been thoroughly tested with Allefresher yet. It is possible to dual-boot CFWs, so if you need a CFW such as ARK-4 for other homebrew, we'd recommend dual-booting PRO C as well until Allefresher is guaranteed to work well on ARK-4.
```

Comment:
The "and" feels a bit weird.

I think something like "Try to use the latest firmware. We recommend PRO C over other CFWs..." works better